servers/extensions/physics_server_2d_extension.h

```diff
@@ -193,6 +193,10 @@ class PhysicsServer2DExtension : public PhysicsServer2D {
 
 	GDVIRTUAL8R_REQUIRED(bool, _body_collide_shape, RID, int, RID, const Transform2D &, const Vector2 &, GDExtensionPtr<Vector2>, int, GDExtensionPtr<int>)
 
+#ifndef DISABLE_DEPRECATED
+	GDVIRTUAL4_COMPAT(_body_set_shape_as_one_way_collision_bind_compat_104736, _body_set_shape_as_one_way_collision, RID, int, bool, real_t)
+#endif
```

Comment:
@dsnopek this PR adds a parameter to a PhysicsServer2DExtension method. I want to be careful about accidentally breaking extensions. Is this the correct way to mitigate this?